lib/cosmos/tools/tlm_viewer/screen.rb

```diff
@@ -23,7 +23,7 @@ class Screen < Qt::MainWindow
     # close_all_screens is called
     @@open_screens = []
 
-    attr_accessor :full_name, :width, :height, :window, :replay_flag
+    attr_accessor :full_name, :width, :height, :window, :replay_flag, :original_target_name
```

Comment:
This is useful to access which target the screen came from

Reply:
agreed